frontend/src/components/SignForms/SignPresenter/SignPresenter.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Box } from '@mui/material';
+
+import { authGreetings, techLogIn, techArtLabel } from './SignPresReceiver';
+import { SignPresenterBox, PresenterWrapper, SighTechArtLabel, SighPresentSubtext } from './styled';
+
+const SignPresenter = (): any => {
+  return (
```

Comment:
any?
There is no need to use `return` statement

Reply:
Fixed